bpf/proxy/proxy.go

```diff
@@ -326,6 +343,26 @@ func (p *proxy) OnEndpointSlicesSynced() {
 	p.forceSyncDP()
 }
 
+func (p *proxy) conntrackCleaner(stopCh <-chan struct{}) {
+	log.Debug("Conntrack cleanup thread started")
+	defer log.Debug("Conntrack cleanup thread stopped")
+
+	ticker := jitter.NewTicker(10*time.Second, 100*time.Millisecond)
+
+	for {
+		select {
+		case <-ticker.C:
+			log.Debug("Conntrack cleanup timer popped")
+			p.dpSyncer.ConntrackScanStart()
+			p.connScan.Scan()
```

Comment:
Worth a comment highlighting the importance of pairing start/end for locking reasons, I think